Services/StakeService/Sources/StakeService.swift

```diff
@@ -39,16 +39,17 @@ public struct StakeService: Sendable {
     
     public func getActiveValidators(assetId: AssetId) throws -> [DelegationValidator] {
         try store.getValidators(assetId: assetId)
```

Comment:
should we call `store.getActiveValidators` to avoid getting unnecessary validators?